Lib/concurrent/futures/process.py

```diff
@@ -180,6 +177,11 @@ def _process_worker(call_queue, result_queue):
             result_queue.put(_ResultItem(call_item.work_id,
                                          result=r))
 
+        # Liberate the resource as soon as possible, to avoid holding onto
```

Comment:
Would it be easy to add a test for this?

Reply:
I added a test for this behavior.